js/packages/common/src/models/metaplex/index.ts

```diff
@@ -90,6 +95,51 @@ export class PayoutTicket {
   }
 }
 
+export class StoreIndexer {
+  key: MetaplexKey = MetaplexKey.StoreIndexerV1;
+  store: StringPublicKey;
+  page: BN;
+  auctionCaches: StringPublicKey[];
+
+  constructor(args: {
+    store: StringPublicKey;
+    page: BN;
+    auctionCaches: StringPublicKey[];
+  }) {
+    this.key = MetaplexKey.StoreIndexerV1;
+    this.store = args.store;
+    this.page = args.page;
+    this.auctionCaches = args.auctionCaches;
+  }
+}
+
+export class AuctionCache {
+  key: MetaplexKey = MetaplexKey.AuctionCacheV1;
+  store: StringPublicKey;
+  timestamp: BN;
+  metadata: StringPublicKey[];
```

Comment:
This is nice. help a lot in rendering the auctions page without querying all nfts associated to the creator.